includes/classes/SVXLink.php

```diff
@@ -469,7 +469,8 @@ private function build_identification() {
 				$id_array['LONG_IDENT_INTERVAL'] = $this->settingsArray['ID_Long_IntervalMin'];
 				$id_array['LONG_VOICE_ID_ENABLE'] = '1';
 				if ($this->settingsArray['ID_Long_AppendTime'] == 'True') {
-					$proc_content .= $this->buildTime();
+					//$proc_content .= $this->buildTime();
+					$id_array['ORP_ANNC_TIME'] = '1';
```

Comment:
This should be correct based on what I have under the similar if statement for custom IDs.